src/Hng.Application/Features/Products/Dtos/CategoryDto.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace Hng.Application.Features.Products.Dtos
+{
+    public class CategoryDto
+    {
+        public Guid Id { get; set; }
+        public string Name { get; set; }
+        public string Description { get; set; }
+        public string Slug { get; set; }
+        public string Parent_id { get; set; }
```

Comment:
Don't do this. Follow pascal case in code but use `JsonPropertyName` attribute to make it snake case